src/locales/en.json

```diff
@@ -2,10 +2,14 @@
   "A store repesents a company or a unique catalog of products. If your OMS is connected to multiple eCommerce stores sellling different collections of products, you may have multiple Product Stores set up in HotWax Commerce.": "A store repesents a company or a unique catalog of products. If your OMS is connected to multiple eCommerce stores sellling different collections of products, you may have multiple Product Stores set up in HotWax Commerce.",
   "Added products to the cycle count out of.": "Added { added } products to the cycle count out of { total }.",
   "ALL": "ALL",
+  "ACCEPTED": "ACCEPTED",
```

Comment:
couldn't we have just styled this to uppercase when needed?

Reply:
Yes, but till now we are following this pattern to use uppercase directly wherever needed.

Even I am having a thought of variable as the key, and the value being the actual string to be used.
Like `add_product_using_csv_success : Added {added} products to the cycle count out of {total}.`.
This way we don't need to make the label changes in the code, and only need to update the locales file whenever required.